core/jazz_t-vault/components/utils/validations.js

```diff
@@ -91,6 +91,24 @@ function validateUserInVaultDeleteInput(event) {
   });
 }
 
+function validateRoleArn(arn) {
+  return new Promise((resolve, reject) => {
+    let regEx = new RegExp("^arn:aws:iam::\\d{12}:role/?[a-zA-Z_0-9+=,.@\\-_/]+");
+    if (regEx.test(arn)) return resolve();
+    else return reject({ "errorType": "inputError", "message": "The provided arn is not valid - " + arn });
+  });
+}
+
+function validateFieldLength(data) {
```

Comment:
Make this generic so it can be used to check for fields of different minimum lengths.
function validateFieldLength(data, minLength)

eg. name is 3 characters
description is 10 characters

Reply:
Also pass just one field instead of all the fields i.e. not data but data.name , data.description separately.

Reply:
Done